MockDuck/Sources/MockBundle.swift

```diff
@@ -15,6 +15,7 @@ final class MockBundle {
 
     var loadingURL: URL?
     var recordingURL: URL?
+    var hitList: [String: Int]?
```

Comment:
I think this name should be more explicit as to what it does... nonIdempotentUrls? Also, maybe a URL instead of a String to offload the library with the url creation responsibility?

Reply:
"nonIdempotent" refers to requests that change some resource on the server. Sometimes we have some Idempotent requests (Like "GET"s) that we want to record several responses for them. I changed it to `repetitiveURLsList` for now, but i think we should find a better name yet.